src/main/resources/layout/style.css

```diff
@@ -414,4 +424,17 @@
     -fx-border-width: 1pt 0 0 0;
     -fx-border-color: LIGHTER_BACKGROUND;
     -fx-padding: 10pt;
+}
+
+.dark-bg {
+    -fx-background-color: BACKGROUND_COLOR;
+    -fx-text-fill: white;
+}
+
+.dark-bg * {
+    -fx-text-fill: inherit;
+}
+
+.text-red {
+    -fx-text-fill: red;
```

Comment:
make sure u have the newest master I also edited this part

Reply:
resolved conflicts with newest master, no changes in style.css were deleted from master

Reply:
> Mutable TREASURE_DISTANCE was not annotated to prevent the preference module from enforcing a fixed distance - thus missing in the mutable section. I've made the mutable dropdown in the plot settings editable to add custom mutables such as TREASURE_DISTANCE.
> Fixes #295

Wouldn't it be a better idea, to leave it, as it is,
but making all preferences selecteable?
Thus, we could add TREASURE_DISTANCE to each Hider/Searcher, we want to give and
if we do not want a fixed TREASURE_DISTANCE, we deselect it?

The Preference objects has to be modified anyway, since issue #322 wants them to have a tooltip